init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/model/Trial.java

```diff
@@ -19,7 +20,7 @@
     @NotEmpty
     private List<@Valid Practitioner> persons;
     @NotEmpty
-    private List<@Valid Site> sites;
+    private List<@Valid SiteDTO> sites;
```

Comment:
you need to re run the build tool schema builder and publish updated schemas in the second repo

Reply:
Will do. Need to discuss how to make validation for the clients models